psen_scan_v2_tutorials/config/scanner2_config.yaml

```diff
@@ -1,7 +1,10 @@
 # %Tag(FULLTEXT)%
-sensor_ip: "192.168.0.20"
+sensor_ip: "192.168.0.20"    # IP address of the laser scanner
+## The following paramters are optional
```

Comment:
I find this misleading, because in the 2-scanner setup you need to change the port numbers.

Reply:
Oh, I wasn't aware of that. Would it be feasible to auto-assign free port(s) as in [that question](https://stackoverflow.com/questions/13734486/using-boostasio-is-there-a-portable-way-to-find-out-free-port-number)?

Reply:
I think it would not work, since we need to bind to two adjacent ports.